README.md

```diff
@@ -0,0 +1,29 @@
+# Masterclasses
+
+Masterclasses are a series of talks and workshops on varied topics presented by the amazing people at Canonical.
+
+Canonical is known for its brilliant people, both in terms of intelligence and varied interests. Often these interests are discovered serendipitously when passing conversations. The initiative would like to provide a platform for this to occur company wide. Recordings will provide longevity to the content and allow people to hone their public speaking and presentational skills.
+
+# Local Development
+
+[masterclassses.canonical.com](masterclasses.canonical.com) is a [Flask v1](https://flask.palletsprojects.com/_/downloads/en/1.1.x/pdf/) app and uses [Flask-base](https://github.com/canonical/canonicalwebteam.flask-base): This is Canoincal's core Flask app that sets default functionality (e.g. redirects.yaml, templates/404.html, robots.txt, favicon, caching headers, security headers).
+
+It uses [dotrun](https://github.com/canonical/dotrun) for local development, defining standard endpoints for `serve`, `build`, `test`, `watch` etc. within the package.json. This allows for a consistent development experience across all Canonical projects.
+
+1. Make a .env.local with the following contents:
+
+```bash
+PRIVATE_KEY_ID=your_private_key_id
+PRIVATE_KEY=your_private_key
+```
+
+these keys are [specs-canonical-com](https://github.com/canonical/masterclasses.canonical.com/blob/3daee9f1bc4232348acc792575c7dd9685f12481/konf/site.yaml#L12) google drive keys.
```

Comment:
Better use a link to the current state in the main branch, than to this specific commit (it might change in the future):

```suggestion
these keys are [specs-canonical-com](https://github.com/canonical/masterclasses.canonical.com/blob/main/konf/site.yaml#L12) google drive keys.
```